tests/responses/test_fileresponse.py

```diff
@@ -80,3 +80,33 @@ def test_unicode_attachment(self):
             response['Content-Disposition'],
             "attachment; filename*=utf-8''%E7%A5%9D%E6%82%A8%E5%B9%B3%E5%AE%89.odt"
         )
+
+    def test_file_to_stream_closes_response(self):
+        # Closing the file_to_stream should call FileResponse.close(), even
+        # when file-like object doesn't have a close() method.
+        class FileLike:
+            def read(self):
+                pass
+
+        class FileLikeWithClose(FileLike):
+            def __init__(self):
+                self.closed = False
+
+            def close(self):
+                self.closed = True
+
+        for filelike_cls in (FileLike, FileLikeWithClose):
+            with self.subTest(filelike_cls=filelike_cls.__name__):
+                filelike = filelike_cls()
+                response = FileResponse(filelike)
+                file_to_stream = response.file_to_stream
+                self.assertFalse(response.closed)
+                # Close file_to_stream.
```

Comment:
Not sure how valuable this comment is, `file_to_stream.close()` seems to explicit about what it does.